README.md

```diff
@@ -1,2 +1,30 @@
 # ami-cleaner
```

Comment:
Change the README.md to a README.rst please. RST is a better markup language, and what I am using across the board now (that Github supports RST files)